pyproject.toml

```diff
@@ -9,14 +9,17 @@ authors = [
     {name = "CFPB", email = "tech@cfpb.gov" }
 ]
 dependencies = [
-    "wagtail>=5.1",
+    "wagtail>=3",
```

Comment:
```suggestion
    "wagtail>=5.2",
```

Reply:
This is fixed here: https://github.com/torchbox-forks/wagtail-sharing/pull/12/commits/7f9718562e4fcd5b11d91a9908641395908c8ebe